documentation/source/index.rst

```diff
@@ -443,6 +451,17 @@ Directive options
 Directive options appear immediately after the directive with no intervening
 blank lines.
 
+``:no-contents-entry:``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+   Tells Sphinx not to add a Table of Contents (ToC) entry for this Dylan object.  This
+   may be especially useful for `dylan:method::`_ directives because sometimes they can
+   unnecessarily clutter the ToC. If the `dylan:method::`_ directives are all colocated
```

Comment:
"Sometimes" refers to the specialization of methods? Are there more cases?

Reply:
The "sometimes" refers to the fact that sometimes individual `method` entries are useful and sometimes they're unnecessary clutter.   The `strings` library comes close to the latter case, because it has a lot of generic functions, all of which have two methods, one for characters and one for strings. The ToC just gets too long.

I can't think of any other case where we would write documentation for an object and then not want it to be in the ToC, but I guess it's possible.